sources/packages/backend/apps/api/src/route-controllers/application/application.students.controller.ts

```diff
@@ -111,9 +111,9 @@ export class ApplicationStudentsController extends BaseController {
         application.data,
       );
     const firstCOEPromise =
-      this.disbursementScheduleService.getFirstDisbursementSchedule({
+      this.disbursementScheduleService.getFirstDisbursementScheduleByApplication(
```

Comment:
await?

Reply:
It is awaited in Promise,all

![image](https://user-images.githubusercontent.com/54600590/211422514-ecea9ecd-7a24-4122-9aef-344e881b9a34.png)